planning/autoware_obstacle_cruise_planner/schema/obstacle_cruise_planner.schema.json

```diff
@@ -0,0 +1,1121 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "autoware_obstacle_cruise_planner parameters",
+  "type": "object",
+  "definitions": {
+    "obstacle_cruise_planner": {
+      "type": "object",
+      "properties": {
+        "common": {
+          "type": "object",
+          "properties": {
+            "planning_algorithm": {
+              "type": "string",
+              "default": "pid_base",
+              "description": "currently supported algorithm is \"pid_base\""
+            },
+            "enable_debug_info": {
+              "type": "boolean",
+              "default": false
```

Comment:
It is missing description field.
Could you add "description" field for all parameters? If you are unsure about what to write, you can keep it empty (`""`), but we need the field since the README rendering is giving us error. 

![image](https://github.com/user-attachments/assets/f5ff29dd-83a1-460c-9b18-12187df5963f)